tripal_chado/api/tripal_chado.custom_tables.api.inc

```diff
@@ -128,6 +128,18 @@ function chado_edit_custom_table($table_id, $table_name, $schema, $skip_if_exist
  */
 function chado_create_custom_table($table, $schema, $skip_if_exists = TRUE,
     $mview_id = NULL, $redirect = TRUE) {
+  
+  if (!$table) {
+    throw new Exception('Please provide a value for the $table argument to the chado_create_custom_table() function'); 
+  }
+  if (!$schema) {
+    throw new Exception('Please provide a value for the $schema argument to the chado_create_custom_table() function');
+  }
+  if ($schema and !is_array($schema)) {
+    throw new Exception('Please provide an array for the $schema argument to the chado_create_custom_table() function');
```

Comment:
Should we be throwing exception outside of entity/field classes? Most of the code using these APIs has not been updated to the `try {} catch{}` method so I worry this is much less friendly to Tripal Admin then the original `tripal_report_error()`. Plus, we could always update `tripal_report_error()` to throw an exception in the future.